java/Nikiforov-Oleksandr/src/com/zoolatech/lecture5/tasks/_2/Task2.java

```diff
@@ -0,0 +1,28 @@
+package com.zoolatech.lecture5.tasks._2;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Create a generic method that accepts a list of values (not only numbers) and returns the max value.
+ */
+
+
+public class Task2 {
+    public static void main(String[] args) {
+        ArrayList<Integer> intlist = new ArrayList<>(Arrays.asList(1, 5, 7, 3, 5));
```

Comment:
1. `intlist` and `strlist` names should be in camelcase: `intList` and `strList`
2. you can define these variables as List<Integer> - then, you can simply assign to them the result of Arrays.asList (without creating of a new ArrayList)